tests/Event/samples/event_with_dummy_location.cdbxml.xml

```diff
@@ -29,7 +29,7 @@
             <cdb:title>'Bekend met Gent' - quiz</cdb:title>
         </cdb:eventdetail>
     </cdb:eventdetails>
-    <cdb:location>
+    <cdb:location cdbid="">
```

Comment:
Why did you add `cdbid=""`? Such cdbXml's never existed. It either had a valid `cdbid` or now `cdbid` property.
This file should not be changed.

Reply:
A commit mistake, I reverted it!